### DISCUSSION TITLE
uc_click() equal for send_keys() or type()?

### DISCUSSION THREAD

**Discussion by caffenie** [seleniumbase/SeleniumBase#2772] [Q&A] [answered]
I have looked through the documentation and the videos but there doesn't seem to be a uc_ equal for type/sending keys function. is there a function or a way to manually mimic it via connect() disconnect() and such?

**Comment by mdmintz** [comment, marked as answer]
You can type text and then immediately call the `reconnect()` function after that for some stealth. I haven't encountered any CAPTCHAs where they're specifically trying to detect you during the typing part. Only the clicking part, which is why there's a `uc_click()` to click in a stealthy way. But if you can find an example for me where typing text gets you detected (where there's no click step for it) then I may add something extra for that in the future.
